### PR TITLE
Add troubleshooting for CephMgrIsAbsent and CephMgrIsMissingReplicas alerts

### PR DESCRIPTION
**Pull request by bklei**
## Summary and Scope

Need steps to repair these alerts (fixed in 1.2)

## Issues and Related PRs

https://connect.us.cray.com/jira/browse/CASMINST-3502

## Testing

hela

### Tested on:

  * hela

### Test description:

Alerts cleared after running these documented steps.

## Risks and Mitigations

No

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md [hunk 1]
```diff
@@ -44,5 +45,36 @@ Alerts for CPUThrottlingHigh on CFS services such as cfs-batcher and cfs-trust c
 
 The CephNetworkPacketsDropped alert does not necessarily indicate there are packets being dropped on an interface on a storage node. In a future release this alert will be renamed to be more generic. If this alert fires, inspect the IP address in the details of the alert to determine the node in question (can be storage, master or worker node). If the interface in question is determined to be healthy, this alert can be ignored.
 
+<a name="cephmgrmissing"></a>
+## CephMgrIsAbsent and CephMgrIsMissingReplicas
 
+If the CephMgrIsAbsent and/or CephMgrIsMissingReplicas alerts fire, use the following steps to ensure the `prometheus` module has been enabled for `Ceph`. The following steps should be executed on ncn-s001:
 
+```bash
```
Comment (nrockershousen) [inline operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:53]: 1. List the enabled modules and look for `prometheus` in the returned output.
    ```bash
    ncn-s001# ceph mgr module ls | jq '.enabled_modules'
    [
      "cephadm",
      "iostat",
      "restful"
    ]
    ```
    
    If `prometheus` is missing from the output, enable it with the following command:
    
    ```bash
    ncn-s001# ceph mgr module enable prometheus
    ```

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md [hunk 2]
```diff
@@ -44,5 +45,36 @@ Alerts for CPUThrottlingHigh on CFS services such as cfs-batcher and cfs-trust c
...
+]
+```
+
+If `prometheus` is missing from the output, enable with the following command:
+
+```bash
+ncn-s001# ceph mgr module enable prometheus
+```
+
+Confirm the module is now enabled:
```
Comment (nrockershousen) [inline operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:68]: 2. Confirm the module is now enabled.

    ```bash
    ncn-s001# ceph mgr module ls | jq '.enabled_modules'
    [
      "cephadm",
      "iostat",
      "prometheus",
      "restful"
    ]
    ```

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md [hunk 3]
```diff
@@ -44,5 +45,36 @@ Alerts for CPUThrottlingHigh on CFS services such as cfs-batcher and cfs-trust c
 
 The CephNetworkPacketsDropped alert does not necessarily indicate there are packets being dropped on an interface on a storage node. In a future release this alert will be renamed to be more generic. If this alert fires, inspect the IP address in the details of the alert to determine the node in question (can be storage, master or worker node). If the interface in question is determined to be healthy, this alert can be ignored.
 
+<a name="cephmgrmissing"></a>
+## CephMgrIsAbsent and CephMgrIsMissingReplicas
 
+If the CephMgrIsAbsent and/or CephMgrIsMissingReplicas alerts fire, use the following steps to ensure the `prometheus` module has been enabled for `Ceph`. The following steps should be executed on ncn-s001:
```
Comment (nrockershousen) [inline operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:51]: `Ceph` should be changed to `ceph` if referring to the command, or Ceph if referring to the service.

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md [hunk 4]
```diff
@@ -44,5 +45,36 @@ Alerts for CPUThrottlingHigh on CFS services such as cfs-batcher and cfs-trust c
...
+ncn-s001# ceph mgr module ls | jq '.enabled_modules'
+[
+  "cephadm",
+  "iostat",
+  "prometheus",
+  "restful"
+]
+```
+
+The CephMgrIsAbsent and CephMgrIsMissingReplicas alerts should now clear in prometheus.
```
Comment (nrockershousen) [inline operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:80]: prometheus -> Prometheus

**Review by nrockershousen — CHANGES_REQUESTED** [review]

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md (hunk 3 above)
Comment (bklei) [reply operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:51]: ok -- i think in this case we're talking generally about the service, so Ceph seems right..

File: operations/system_management_health/Troubleshoot_Prometheus_Alerts.md (hunk 4 above)
Comment (bklei) [reply operations/system_management_health/Troubleshoot_Prometheus_Alerts.md:80]: good catch, will change

**Pull request merged**